python/ql/test/library-tests/ControlFlow/successors/RaisingFlow.ql

```diff
@@ -0,0 +1,17 @@
+/**
+ * @name Raising Flow
+ * @description Test
+ */
+
+import python
+
+from ControlFlowNode p, ControlFlowNode s, string kind
+where p.getASuccessor() = s and
+(if s = p.getAnExceptionalSuccessor() then
+    kind = "exception"
+ else
+    kind = "  normal "
+) and
+not p.getNode() instanceof Scope and 
+not s.getNode() instanceof Scope
+select p.getNode().getLocation().getStartLine(), p.toString(), kind, s.getNode().getLocation().getStartLine(), s
```

Comment:
I see that you've made it so that the `kind` string has a constant size, but I'm not sure how that is helpful (seeing as `p.toString()` can have varying size). Would it make sense to put `kind` before `p.toString()`?

Reply:
That test shouldn't be there at all as it is a useless duplicate of `library-tests/ControlFlow/raising_stmts/RaisingFlow.ql`
Removed now.